product_logistics_uom/models/product_template.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Akretion (https://www.akretion.com).
+# @author Raphaël Reverdy <raphael.reverdy@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    volume_uom_id = fields.Many2one(
+        "uom.uom",
+        string="Volume Unit of Measure",
+        domain=lambda self: [
+            ("category_id", "=", self.env.ref("uom.product_uom_categ_vol").id)
+        ],
+        default=lambda x: x._get_volume_uom_id_from_ir_config_parameter(),
```

Comment:
Why use the x instead of self? I prefer not to use one character variables.

Reply:
Done